doc/HACKING.md

```diff
@@ -0,0 +1,27 @@
+# Hacking on the kata-operator
+
+## Using a custom kata-operator-payload image
+
+To set a custom image create a configmap. Open the file deploy/configmap_payload.yaml and
+change
+
+    daemon.payload: quay.io/<username>/mykatapayload:mytag
+
+## How to create a custom payload container image
+
+Based on an existing and known to work set of RPMs it is possible to replace
+packages.
+
+Note: it is not possible to add additonal RPMs this way
+
+An example:
+
+1. skopeo copy docker://quay.io/jensfr/kata-operator-payload:4.6.0 oci:/tmp/kata-operator-payload:4.6.0
+2. oci-image-tool unpack --ref name=4.6.0  /tmp/kata-operator-payload kata-operator-payload-unpacked-4.6.0
+3. cp -r /tmp/kata-operator-payload-unpacked-4.6.0/packages $KATA_OPERATOR_REPO/images/payload
+4. cd $KATA_OPERATOR_REPO/images/payload
+5. replace RPMs in packages/ with custom RPMs
+6. podman build --no-cache -f Dockerfile.custom quay.io/<username>/mykatapayload:mytag
```

Comment:
can it also be built with docker?

Reply:
what's docker? ;-) Yes, I  see no reason why not.